src/styles/styles.scss

```diff
@@ -37,6 +36,62 @@ html, body {
   height: 100%;
 }
 
+.index_bottom_container {
```

Comment:
could you please make this a component? This can be called a hero-section component and add all the styles inside there. We want to follow a component based design with a mobile first approach.

Reply:
sure will do!